client/core/trade.go

```diff
@@ -1562,6 +1562,7 @@ func (t *trackedTrade) isRedeemable(ctx context.Context, match *matchTracker) (r
 				if match.MetaData.Proof.SelfRevoked {
 					return false, false // already self-revoked
 				}
+				// Here we can check to see if this is a redeem we failed to record...
```

Comment:
This comment pertains to https://github.com/decred/dcrdex/pull/2049#discussion_r1069576868
It's unrelated to bonds, but it snuck in.